lib/gs-plugin-loader.c

```diff
@@ -457,6 +457,12 @@ gs_plugin_error_handle_failure (GsPluginLoaderHelper *helper,
 		return TRUE;
 	}
 
+        /* this is only ever informational */
```

Comment:
Is this patch already upstream? If not: the indentation doesn't match the surrounding code.

Reply:
Yeah... is this actually a cherry pick or is it copy/pasted?! It doesn't match https://gitlab.gnome.org/GNOME/gnome-software/commit/62f6314f9f1262ac7b3c5d78bb96fe57240f7d4f